sqlmesh/dbt/manifest.py

```diff
@@ -0,0 +1,224 @@
+                ),
+                path=Path(macro.original_file_path),
+            )
+
+    def _load_models_and_seeds(self) -> None:
+        for node in self._manifest.nodes.values():
+            if node.resource_type not in ("model", "seed"):
+                continue
+
+            macro_references = _macro_references(self._manifest, node)
+
+            node_dict = node.to_dict()
+            node_dict.pop("database", None)
+            base_config = {**_config(node), **node_dict, "path": Path(node.original_file_path)}
+
+            if node.resource_type == "model":
+                self._models_per_package[node.package_name][node.name] = ModelConfig(
+                    sql=node.raw_code,
+                    dependencies=Dependencies(
+                        macros=macro_references,
+                        refs=_refs(node),
+                        sources=_sources(node),
+                    ),
+                    **base_config,
+                )
+            else:
+                self._seeds_per_package[node.package_name][node.name] = SeedConfig(
+                    dependencies=Dependencies(macros=macro_references),
+                    **base_config,
+                )
+
+    @property
+    def _manifest(self) -> Manifest:
+        if not self.__manifest:
+            self.__manifest = self._load_manifest()
+        return self.__manifest
+
+    def _load_manifest(self) -> Manifest:
+        do_not_track()
+
+        args: Namespace = Namespace(
+            vars={} if DBT_VERSION >= (1, 5) else "{}",
+            profile=self.profile_name,
+            profiles_dir=str(self.profiles_path),
+            target=self.target,
+            macro_debugging=False,
+        )
+        flags.set_from_args(args, None)
+
+        profile = self._load_profile()
+        project = self._load_project(profile)
+        runtime_config = RuntimeConfig.from_parts(project, profile, args)
+
+        self._project_name = project.project_name
+
+        register_adapter(runtime_config)
+        manifest = ManifestLoader.get_full_manifest(runtime_config)
+        reset_adapters()
+        return manifest
+
+    def _load_project(self, profile: Profile) -> Project:
+        project_renderer = DbtProjectYamlRenderer(profile)
+        return Project.from_project_root(str(self.project_path), project_renderer)
+
+    def _load_profile(self) -> Profile:
+        profile_renderer = ProfileRenderer({})
+        raw_profiles = read_profile(str(self.profiles_path))
+        return Profile.from_raw_profiles(
+            raw_profiles=raw_profiles,
+            profile_name=self.profile_name,
+            renderer=profile_renderer,
+            target_override=self.target,
+        )
+
+
+def _config(node: t.Union[ManifestNode, SourceDefinition]) -> t.Dict[str, t.Any]:
+    rendered_config = node.config.to_dict()
+    unrendered_config = node.unrendered_config
+    return {k: v for k, v in rendered_config.items() if k in unrendered_config}
```

Comment:
What problem did you run into that we don't want all rendered config?

Reply:
the `config` attribute contains all fields, both specified and default. We only want to pick the overridden ones and inherit everything else from the node itself.

Reply:
What is an example where the inherited behavior differs from the rendered config?

Reply:
```
    "model.sushi.waiter_revenue_by_day": {
      "database": "main",
      "schema": "sushi",
      "name": "waiter_revenue_by_day",
      "resource_type": "model",
      "package_name": "sushi",
      "path": "waiter_revenue_by_day.sql",
      "original_file_path": "models/waiter_revenue_by_day.sql",
      "unique_id": "model.sushi.waiter_revenue_by_day",
      "fqn": [
        "sushi",
        "waiter_revenue_by_day"
      ],
      "alias": "waiter_revenue_by_day",
      "config": {
        "enabled": true,
        "alias": null,
        "schema": null,
        "database": null,
 ```

Reply:
updated this as per our conversation